backend/core/src/email/email.service.ts

```diff
@@ -224,34 +255,115 @@ export class EmailService {
     )
   }
 
-  public async listingOpportunity(user: User, appUrl: string) {
-    const jurisdiction = await this.getUserJurisdiction(user)
-    void (await this.loadTranslations(jurisdiction, user.language))
+  public async listingOpportunity(listing: Listing) {
+    const jurisdiction = await this.jurisdictionService.findOne({
+      where: {
+        id: listing.jurisdiction.id,
+      },
+    })
+    void (await this.loadTranslations(jurisdiction, Language.en))
     const compiledTemplate = this.template("listing-opportunity")
 
     if (this.configService.get<string>("NODE_ENV") == "production") {
       Logger.log(
-        `Preparing to send a listing opportunity email to ${user.email} from ${jurisdiction.emailFromAddress}...`
+        `Preparing to send a listing opportunity email for ${listing.name} from ${jurisdiction.emailFromAddress}...`
```

Comment:
suggestion: maybe dropping the trailing `...`

Reply:
This is the pattern in the other loggers here. However, I don't think I have ever seen these logs in the logs before.